packages/app/src/components/Layout/Tooltip.js

```diff
@@ -81,12 +81,37 @@ export class Tooltip extends React.Component {
         </li>
     )
 
-    renderItems = itemDisplayNames =>
-        itemDisplayNames.map(name => (
-            <li key={`${this.props.dimensionId}-${name}`} style={styles.item}>
-                {name}
-            </li>
-        ))
+    renderItems = itemDisplayNames => {
+        const renderLimit = 5
+
+        const itemsToRender = itemDisplayNames
+            .slice(0, renderLimit)
+            .map(name => (
+                <li
+                    key={`${this.props.dimensionId}-${name}`}
+                    style={styles.item}
+                >
+                    {name}
+                </li>
+            ))
+
+        if (itemDisplayNames.length > renderLimit) {
+            itemsToRender.push(
+                <li
+                    key={`${this.props.dimensionId}-render-limit`}
+                    style={styles.item}
+                >
+                    {`And ${itemDisplayNames.length - renderLimit} ${
```

Comment:
Translations are missing here, and this won't work properly for all languages.
1) The string should not contain the number, that should be interpolated.
    The problem is that in some languages the position of the number is not the same as for English.
2) plurals are tricky:
    https://www.i18next.com/translation-function/plurals